### PR TITLE
Namespace Specific ValidationFailureAction

### LINKED ISSUE #410: [Docs] Namespace Specific ValidationFailureAction
**Related Issue**
- kyverno/kyverno#559

**Related PR**
- kyverno/kyverno#2794

### PR DESCRIPTION
**Pull request by ojhaarjun1**
Closes #410 

Signed-off-by: Kumar Mallikarjuna <kumarmallikarjuna1@gmail.com>

### CODE REVIEW

File: content/en/docs/Writing policies/validate.md [hunk 1]
```diff
@@ -81,6 +81,40 @@ Change the `development` value to `production` and try again. Kyverno permits cr
...
+    - action: enforce
+      namespaces:
+        - default
+    - action: audit
+      namespaces:
+        - test
+  rules:
+    - name: check-label-app
+      match:
+        resources:
```
Comment (chipzoller) [inline content/en/docs/Writing policies/validate.md:105]: Let's add `any` here since this is the new standard.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Inline comments

**Comment by chipzoller** [comment]
Can we close this out, please?

File: content/en/docs/Writing policies/validate.md [hunk 2]
```diff
@@ -81,6 +81,40 @@ Change the `development` value to `production` and try again. Kyverno permits cr
...
+          - Pod
+      validate:
+        message: "The label `app` is required."
+        pattern:
+          metadata:
+            labels:
+              app: "?*"
+```
+
+In the above policy, for Namespace `default`, validation failure action is set to `enforce` and for Namespace `test`, it's set to `audit`. For all other Namespaces, the action defaults to the `validationFailureAction` field.
```
[the remark this answers is not in the document]
Comment (ojhaarjun1) [reply content/en/docs/Writing policies/validate.md:116]: In this context, "validation failure action" refers to the action and not the attribute `validationFailureAction`. Writing `validationFailureAction` would instead imply changing the attribute value internally based on the namespace.

**Comment by ojhaarjun1** [comment]
@chipzoller - Added inline comments to the example YAML. Is this what you meant?

**Review by chipzoller — APPROVED** [review]

**Pull request merged**